labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -99,7 +99,10 @@ object ListManipulationExercise01 {
    * Implement it whatever way suites you best. Hint: it can be done in a neat way using recursion. 
    */
   def tails[T](l: List[T]): List[List[T]] = {
-    error("fix me")
+    l match {
+      case Nil => List(Nil)
+      case _ :: xs => l :: (tails(xs))
```

Comment:
I saw xs used as a variable in Scala a lot - presumably, according to some convention. What does it stand for?